src/style.css

```diff
@@ -1 +1,115 @@
-/* add styles here */
+.html {
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: Avenir;
+  src:
+    url(./fonts/Avenir-Book.ttf) format('Avenir'),
+    url(./fonts/Avenir-Heavy.ttf) format('Avenir'),
+    url(./fonts/Avenir.ttc);
```

Comment:
The `format()` function in the `@font-face` rule is meant to specify the format of the font file being linked, not the name of the font. Each `url()` should be followed by a `format('truetype')`, `format('woff')`, etc., corresponding to the format of the linked font file.